src/lib/axes.typ

```diff
@@ -55,6 +55,10 @@
 
 // Format a tick value
 #let format-tick-value(value, tic-options) = {
+  // Without it we get negative zero in conversion
+  // to content! Typst has negative zero floats.
+  if value == 0 { value = 0 }
```

Comment:
wait shouldn't this be checking for `-0.0` then? Maybe add a test for this?

Reply:
It does: `-0 == 0`… there is no way to detect `-0`, because by IEE definition it must compare equal to 0 (you can only know by checking the sign bit). I was complaining on Discord that Typst should never format `-0` as `0`.